pandas/core/generic.py

```diff
@@ -1611,7 +1611,7 @@ def __round__(self: NDFrameT, decimals: int = 0) -> NDFrameT:
     # have consistent precedence and validation logic throughout the library.
 
     @final
-    def _is_level_reference(self, key, axis=0):
+    def _is_level_reference(self, key: Hashable, axis=0) -> bool_t:
```

Comment:
I'm never sure whether they can also be Sequence[Hashable] in case of a MultiIndex

Reply:
im pretty sure it shouldn't be